### DISCUSSION TITLE
Windows App SDK 1.3.2 (1.3.230602002)

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#3674] [Announcements]
## Windows App SDK 1.3.2 (1.3.230602002)

Windows App SDK 1.3.2 is a stable servicing release that fixes critical bugs for 1.3. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.3 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://learn.microsoft.com/windows/apps/windows-app-sdk/stable-channel#version-132-13230602002) for the list of issues fixed in this release.

### Updating to this version

The 1.3.2 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.3.230602002

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.3.2'>Windows App SDK 1.3.2 (1.3.230602002)</a>.</em>

**Comment by ghost1372** [comment]
Where is 1.4-preview 1 discussions?
Nuget package was released a few days ago
